core/sr-primitives/src/lib.rs

```diff
@@ -40,6 +40,10 @@ pub mod testing;
 pub mod traits;
 pub mod generic;
 pub mod transaction_validity;
+pub mod doughnut {
+	//! Runtime doughnut types
+	pub use doughnut_rs::v0::parity::DoughnutV0;
```

Comment:
I need to check doughnut_rs code, but why there is module called `parity`?

Reply:
Parity codec